scripts/properties2json.js

```diff
@@ -1,47 +1,74 @@
-var properties = require(`properties-parser`);
-var write = require(`fs-writefile-promise`);
-var path = require(`path`);
-var FS = require(`q-io/fs`);
-var src = `locales`;
+var config = require('../intl-config.js');
```

Comment:
This was previously an unused file in the repo that was being prepped for l10n usage at one point.

It was previously a copy of an older version of the same file from donate. I just updated it to the newer version from donate.

It turns the contents of the .properties files into .json files so they can be better read by js.